<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Banc de Test Carte ARAL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_panel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_panel_ACK"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ARRET / Affichage du Bilan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton_arret">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Commencer / Reprise du test en boucle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton_repriseTest">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre de tours de tests</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_nbTours">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton_nbTours">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton_lancementTestNuit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>LANCER UN TEST DE 8H</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton_activer_filtrage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Activer Filtrage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton_reglage_nb_etat_en_test">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Mettre le mode avec 2 Ã©tats, Court-circuit et Alarme</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichiers">
    <property name="title">
     <string>Fichiers</string>
    </property>
    <addaction name="actionTableau_Voies_en_Cours"/>
    <addaction name="actionTableau_Voies_Bilan"/>
    <addaction name="separator"/>
    <addaction name="action_reglage_du_switch_SW2"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCreation_PDF">
    <property name="title">
     <string>Creation PDF</string>
    </property>
    <addaction name="actionFicheValidation"/>
   </widget>
   <widget class="QMenu" name="menuPort_COM">
    <property name="title">
     <string>Port COM</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFichiers"/>
   <addaction name="menuPort_COM"/>
   <addaction name="menuCreation_PDF"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClearLog"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionTableau_Voies_en_Cours"/>
   <addaction name="separator"/>
   <addaction name="actionFicheValidation"/>
  </widget>
  <action name="actionTableau_Voies_Bilan">
   <property name="text">
    <string>Tableau Voies Bilan</string>
   </property>
  </action>
  <action name="actionTableau_Voies_en_Cours">
   <property name="text">
    <string>Tableau Voies en Cours</string>
   </property>
  </action>
  <action name="actionFicheValidation">
   <property name="text">
    <string>Fiche de Validation</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear &amp;Log</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionactionSetMainWindowIcon">
   <property name="icon">
    <iconset resource="Ressources.qrc">
     <normaloff>:/images/logo.ico</normaloff>:/images/logo.ico</iconset>
   </property>
   <property name="text">
    <string>actionSetMainWindowIcon</string>
   </property>
  </action>
  <action name="action_reglage_du_switch_SW2">
   <property name="icon">
    <iconset>
     <normaloff>:/images/switch.png</normaloff>:/images/switch.png</iconset>
   </property>
   <property name="text">
    <string>RÃ©glage du switch SW2</string>
   </property>
   <property name="toolTip">
    <string>switch_SW2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
